.github/workflows/test.yaml

```diff
@@ -22,9 +22,11 @@ jobs:
           curl -H "Accept: application/vnd.github+json" -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" -L -o $ARTIFACT_RELEASE_ARCHIVE_NAME $ARTIFACT_RELEASE_ARCHIVE_LINK
           unzip $ARTIFACT_RELEASE_ARCHIVE_NAME
           tar -xvf $RELEASE_ARCHIVE_NAME
-      
-      - name: Add Cairo compiler to path
-        run: echo "$(pwd)/cairo/bin" >> $GITHUB_PATH
-      
+
+      - name: Check Cairo compiler version and add to path
+        run: |
+          $(pwd)/cairo/bin/cairo-compile --version
```

Comment:
Version gets printed [here](https://github.com/auditless/cairo-template/actions/runs/4359589167/jobs/7621534668#step:4:10)
<img width="435" alt="image" src="https://user-images.githubusercontent.com/35031356/223580716-4db4655c-7460-4ded-bf2d-280f988f5a0f.png">

Reply:
Thanks for making this easy to review!